docs/components/gantry/single-axis.md

```diff
@@ -59,11 +59,82 @@ The following attributes are available for `single-axis` gantries:
 | --------- | ---- | --------- | ----------  |
-| `length_mm` | int | **Required** | Length of the axis of the gantry in millimeters. |
-| `motor` | string | **Required** | `name` of the [motor](/components/motor/) that moves the gantry's actuator. |
+| `length_mm` | int | **Required** | The length of the axis of the gantry in millimeters. |
+| `motor` | string | **Required** | The `name` of the [motor](/components/motor/) that moves the gantry's actuator. |
 | `axis` | object | **Required** | The translational axis for the gantry. Must be exactly one of `x`, `y`, or `z`. |
-| `board`  |  Optional | string | `name` of the [board](/components/board/) containing the limit switches and pins. If `limit_pins` exist, `board` is required. |
-| `limit_pins`  | Optional | object | The `boards`'s pins attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), `limit_pins` are required to be set. |
-| `limit_pin_enabled_high` | boolean | Optional | Whether the limit pins are enabled. <br> Default: `false` |
+| `board`  |  string | Optional | The `name` of the [board](/components/board/) containing the limit switches and pins. If `limit_pins` is set, `board` is **required**. |
+| `limit_pins`  | object | Optional | The pins on the `board` which are attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), you must set `limit_pins`. The order of these pins is important. If the pins are in the wrong order, the gantry may try to travel beyond the limit switch (or switches). The switch representing the home (`0`) position must be first in the list. In addition, if you are configuring multiple `single-axis` gantries, you can [specify the order of homing routines](#specify-the-order-of-homing-routines) if needed. |
+| `limit_pin_enabled_high` | boolean | Optional | Whether the limit pin must be “high” or “low” to be considered “hit”, with a value of `true` representing "high". This attribute is **required** if `limit_pins` is set.<br> Default: `false` |
 | `mm_per_rev` | int | Optional | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
 | `gantry_rpm` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
+
+## Specify the order of homing routines
+
+When `limit_pins` is set, a `single-axis` gantry component performs a homing routine when `viam-server` starts.
+The axis will move towards one limit switch, then the other, before stopping at 80% of the travel of the axis.
+
+If there are multiple `single-axis` gantries configured (for example, when using a [`multi-axis` gantry](/components/gantry/multi-axis/)), the order in which these homing routines are executed may be important.
```

Comment:
```suggestion
If you configure multiple `single-axis` gantries (for example, when using a [`multi-axis` gantry](/components/gantry/multi-axis/)), the order in which these homing routines are executed may be important.
```
[nit] speak to user. Also, when it says "...may be important," can we be more clear about whether it is or isn't?

Reply:
maybe something like "If you configure multiple `single-axis` gantries into a `multi-axis` gantry, the axes will home in the order they are listed in the `multi-axis` gantry"